inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/SinkFieldBase.java

```diff
@@ -62,4 +62,7 @@ public class SinkFieldBase {
     @ApiModelProperty("Field order")
     private Short rankNum;
 
+    @ApiModelProperty("Column family for columnar storage e.g.hbase")
+    private String columnFamily;
```

Comment:
Do we really need to add every specified field to the table when using a different type of sink?

Reply:
This happens to iceberg table specified fields and i just followed the pattern.